src/SonarScanner.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -354,6 +364,8 @@
       <SonarReportFilePath Condition= "$(SonarErrorLogExists) == 'true'" Include="$(SonarErrorLog)" />
     </ItemGroup>
 
+    <WriteSonarTelemetry Filename="$(SonarProjectTelemetryFilePath)" Key="dotnetenterprise.s4net.build.target_framework" Value="$(TargetFrameworkMoniker)" />
```

Comment:
```suggestion
    <WriteSonarTelemetry Filename="$(SonarProjectTelemetryFilePath)" Key="dotnetenterprise.s4net.build.target_framework_moniker" Value="$(TargetFrameworkMoniker)" />
```

Reply:
I might have made a mess with the rebase. Sorry for that.